### ISSUE TITLE
no registration link via email ?!?

### ISSUE THREAD

**Issue by Tom-moves** [LAION-AI/Open-Assistant#2466]
hello,
after signing up via email I don't receive any registration email.
Tried it several times, but it does not work. There is no email in the spam folder either.

Help is highly appreciated.
Thank you!

**Comment by CloseChoice** [comment]
Okay, let's have the discussion in this issue:
@Tom-moves could you please sign up with a different email and check if that works. Also what's the domain of your email?

**Comment by Tom-moves** [comment]
thanks for the respond!
my email is: 
grace-thomas@t-online.de
that is the only one, that I use and it works as I just got the github mails

**Comment by CloseChoice** [comment]
I'm not doubting that your email is working but for some reasons mails from openassistant don't make it into your inbox. I just created a 10 minutemail email address and signing up for OA works fine with it. Would it be an option for you to use such a throw-away mail for OA?

**Comment by CloseChoice** [comment]
yep, that works. But as soon as the mail expires you have no possibility to regain access to your account. Also as of today you cannot change the email of the account

**Comment by Tom-moves** [comment]
so that means: I log in right now and stay in x hours in order to try to change the email of the account later on?

**Comment by Tom-moves** [comment]
I appreciate your help and guess I understand. But obviously I am interested in a lasting solution to try it out.
So there is no other possibility to get an lasting access to OA for me?

**Comment by CloseChoice** [comment]
yep, just create another email and sign up with that

**Comment by Tom-moves** [comment]
Ok thank you very much CloseChoice! I´ll try to do that.

**Comment by CloseChoice** [comment]
I'll close this issue for now. If it still doesn't work, please just comment in here, and I'll reopen it.

**Comment by nguyenhoan1988** [comment]
The signin email from Open Assistant is classified as Spam by Gmail.

The problem might be:
1. the sender address, I got the email sent by `open-assistent@laion.ai`, with a typo
2. the content of the email: there is little information regarding the project/sender/organization which will make the email suspicious.

**Comment by RichardScottOZ** [comment]
Yes, just saw this happen.

**Comment by Tom-moves** [comment]
I tried to use it through discord which works. Thanks.
